### PR TITLE
Bump Eirini to 0.0.21

### PR DESCRIPTION
**Pull request by kramerul**
## Description

Using latest eirini release.

## Test plan

Install kubecf with latest cf-operator. Run `cf push` afterwards.

## Problems

`cf push` is not working. But this is related to https://github.com/cloudfoundry-incubator/cf-operator/issues/648

### CODE REVIEW

File: deploy/helm/scf/assets/operations/instance_groups/eirini.yaml [hunk 1]
```diff
@@ -106,6 +117,15 @@
           cc_cert: ((cc_bridge_tps.certificate))
           cc_key: ((cc_bridge_tps.private_key))
           cc_ca: ((service_cf_internal_ca.certificate))
+        quarks:
+            ports:
+            - name: http
+              protocol: TCP
+              internal: 8085
```
Comment (phil9909) [inline deploy/helm/scf/assets/operations/instance_groups/eirini.yaml]:
```suggestion
```

File: deploy/helm/scf/assets/operations/instance_groups/eirini.yaml [hunk 2]
```diff
@@ -106,6 +117,15 @@
           cc_cert: ((cc_bridge_tps.certificate))
           cc_key: ((cc_bridge_tps.private_key))
           cc_ca: ((service_cf_internal_ca.certificate))
+        quarks:
+            ports:
+            - name: http
+              protocol: TCP
```
Comment (phil9909) [inline deploy/helm/scf/assets/operations/instance_groups/eirini.yaml]:
```suggestion
```

File: deploy/helm/scf/assets/operations/instance_groups/eirini.yaml [hunk 3]
```diff
@@ -106,6 +117,15 @@
           cc_cert: ((cc_bridge_tps.certificate))
           cc_key: ((cc_bridge_tps.private_key))
           cc_ca: ((service_cf_internal_ca.certificate))
+        quarks:
+            ports:
+            - name: http
```
Comment (phil9909) [inline deploy/helm/scf/assets/operations/instance_groups/eirini.yaml]: This (the http port) should not be needed anymore.
```suggestion
```

**Comment by f0rmiga** [comment]
@kramerul The bump to 0.0.21 seems good. I tested with cf-operator on commit https://github.com/cloudfoundry-incubator/cf-operator/commit/c193b5b616ef22a3a65489e4ff573109480a5c62, which is part of https://github.com/cloudfoundry-incubator/cf-operator/pull/661, and the failure I get is due to the `opi-task-downloader` init container not being able to connect to blobstore.service.cf.internal, which makes perfect sense given that it is in a different namespace and it's not using our aliases on the CF namespace. I pushed a commit on the PR while it was on the SCF side that fixed this. I'll accept this PR as is and I'll push a fix for that in a separate PR.

**Pull request merged**